cmake/Hunter/config.cmake

```diff
@@ -17,3 +17,9 @@ hunter_config(libsecp256k1
     SHA1 179e316b0fe5150f1b05ca70ec2ac1490fe2cb3b
     CMAKE_ARGS SECP256K1_BUILD_TEST=OFF
     )
+
+hunter_config(sr25519
+    URL https://github.com/Harrm/sr25519-crust/archive/master.zip
```

Comment:
This will break as soon as you change master, but the latest commit seems fine:
```
    URL https://github.com/Harrm/sr25519-crust/archive/a598ed810c806b533991d9f229855e5ec0cdcd0c.tar.gz
    SHA1 966cbc88c9a5e31c9aa3fb0530acea2a402f249d
```